lib/matplotlib/tests/test_axes.py

```diff
@@ -1647,7 +1647,7 @@ def test_contour_hatching():
 
     fig = plt.figure()
     ax = fig.add_subplot(111)
-    cs = ax.contourf(x, y, z, hatches=['-', '/', '\\', '//'],
+    cs = ax.contourf(x, y, z, hatches=['/', '\\', '//', '-'],
```

Comment:
Could we mpl20 and remove-text=True this example while we are changing the image?